charts/argo-cd/values.yaml

```
@@ -213,6 +213,9 @@ dex:
       labels: {}
     serviceMonitor:
       enabled: false
+      # selector:
+      #   prometheus: kube-prometheus
+      # additionalLabels: {}
       interval: 30s
```

Also add namespace and use sort them to be consistent across all components
```suggestion
      interval: 30s
      # selector:
      #   prometheus: kube-prometheus
      # namespace: monitoring
      # additionalLabels: {}
```